scripts/toolboxSrc/postDeploy.ts

```diff
@@ -1,10 +1,35 @@
 import fs from 'fs'
 import path from 'path'
 import getRethink from '../../packages/server/database/rethinkDriver'
-import db from '../../packages/server/db'
 import getProjectRoot from '../webpack/utils/getProjectRoot'
+import getRedis from '../../packages/server/utils/getRedis'
 
 const PROJECT_ROOT = getProjectRoot()
+const redis = getRedis()
```

Comment:
nit: doing this here opens the connection pool when the file is imported vs. when a function is called.
ideally, files are side-effect free, but functions do things (the exception being singletons)